lib/array.js

```diff
@@ -1,42 +1,76 @@
 'use strict';
 
 const common = require('@metarhia/common');
+const { asyncIter } = require('./async-iterator.js');
+
+const copyClass = name => {
+  switch (name) {
+    case 'Set':
+      return Set;
+    case 'Map':
+      return Map;
+    case 'Array':
+      return Array;
+    case 'String':
+      return String;
+  }
+  return null;
+};
 
-// Asynchronous map (iterate parallel)
```

Comment:
Why did you delete comments?

Reply:
The same question goes for other deleted comments.

Reply:
I replaced them, because I have already comments to function
``` javascript
const map = (
  // Asynchronous map (iterate parallel)
  items, // array, incoming
  fn, // function, (current, callback) => callback(err, value)
  //   to be executed for each value in the array
  //   current - current element being processed in the array
  //   callback - function(err, value)
  done // function (optional), on done callback function(err, result)
) => {
```
and the same with other functions.

Reply:
This is the old commenting style and it is not used anymore. Please, consider checking out the new commenting style [here](https://github.com/metarhia/metadoc).